include/deal.II/base/point.h

```diff
@@ -358,6 +366,25 @@ inline Point<dim, Number>::Point(const Number x, const Number y, const Number z)
 }
 
 
+
+template <int dim, typename Number>
+inline Point<dim, Number>::Point(
+  const boost::geometry::model::
+    point<Number, dim, boost::geometry::cs::cartesian> boost_pt)
+{
+  this->values[0]                = boost::geometry::get<0>(boost_pt);
```

Comment:
I should have mentioned this before, but can you add
```
  Assert (dim<=3, ExcNotImplemented());
```
here? It's not inconceivable that someone may want to use this function for higher dimensional points, but the function is then not correct.

Reply:
Thank you @bangerth ... the last days I passed some long hours on a code I am developing for a project, I hope to have time this weekend to fix all the minor bugs / comments made here and in my other PRs (hopefully I'll also make a new one)!